tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -27,6 +29,14 @@ setup() {
     unencrypted_image_1="quay.io/sjenning/nginx:1.15-alpine"
     unencrypted_image_2="quay.io/prometheus/busybox:latest"
     large_image="quay.io/confidential-containers/test-images:largeimage"
+    AUTHENTICATED_IMAGE="${AUTHENTICATED_IMAGE:-quay.io/kata-containers/confidential-containers-auth:test}"
+    AUTHENTICATED_IMAGE_USER=${AUTHENTICATED_IMAGE_USER:-}
+    AUTHENTICATED_IMAGE_PASSWORD=${AUTHENTICATED_IMAGE_PASSWORD:-}
+
+     # Add the kubernetes imagePullSecret for the nydus-snapshotter metadata pull
+    kubectl delete secret cococred --ignore-not-found
+    kubectl create secret docker-registry cococred --docker-server="https://quay.io/kata-containers/confidential-containers-auth" \
```

Comment:
`--docker-server="echo "$AUTHENTICATED_IMAGE"  | cut -d':' -f1"`

Reply:
Oops - you can spot that I wrote it all badly and then refactored (most of it)